### PR TITLE
Privacy settings update #602

### PR DESCRIPTION
**Pull request by Ezyweb-uk**
Proposed updates for issue #602

### CODE REVIEW

**Comment by peter-mw** [comment]
Hi Nick, thanks for the updates, its merged now into dev branch

**Comment by Ezyweb-uk** [comment]
That was quick - Thanks Peter!

File: userfiles/modules/users/terms/index.php [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
+    $terms_label = 'I agree with the <a href="' . site_url() . 'terms" target="_blank">Terms and Conditions</a>';
 }
 ?>
-<div class="row" style="margin-top: 20px;margin-bottom: 20px;">
+<div class="row" style="margin-top: 10px;margin-bottom: 10px;">
     <div class="col-xs-12" class="i_agree_with_terms_row_popup">
         <div class="form-group">
             <div class="custom-control custom-checkbox">
-                <h5><?php _e('You must agree to terms and conditions to continue'); ?></h5>
+                <!-- <h5><?php _e('You must agree to the Terms and Conditions to continue'); ?></h5> //-->
```
Comment (peter-mw) [inline userfiles/modules/users/terms/index.php]: this must be not commented out

Comment (Ezyweb-uk) [reply userfiles/modules/users/terms/index.php]: Might be nice to have a configurable option to also show a newsletter subscribe checkbox similar to the example [here](https://www.formstack.com/templates/gdpr-compliant-contact-form), and allow the default text to be edited (similar to the terms consent text)?

**Pull request merged**